scale/ingest/models.py

```diff
@@ -296,44 +297,22 @@ def start_ingest_tasks(self, ingests, scan_id=None, strike_id=None):
         :param strike_id: ID of Strike that generated ingest
         :type strike_id: int
         """
-
-        # Create new ingest job and mark ingest as QUEUED
-        ingest_job_type = Ingest.objects.get_ingest_job_type()
-
+        
+        messages = []
         for ingest in ingests:
             logger.debug('Creating ingest task for %s', ingest.file_name)
-
-            when = ingest.transfer_ended if ingest.transfer_ended else now()
-            desc = {'file_name': ingest.file_name}
-
+            
             if scan_id:
-                # Use result from query to get ingest ID
-                # We need to find the id of each ingest that was created.
-                # Using scan_id and file_name together as a unique composite key
-                ingest_id = Ingest.objects.get(scan_id=ingest.scan_id, file_name=ingest.file_name).id
-
-                desc['scan_id'] = scan_id
-                event = TriggerEvent.objects.create_trigger_event('SCAN_TRANSFER', None, desc, when)
+                # TODO: Need to make sure scans work
```

Comment:
We still need to check this?

Reply:
Correct. I've been focusing on getting strikes functional then we'll worry about scans.